### ISSUE TITLE
Syntax color

### ISSUE THREAD

**Issue by klingberg** [luishdez/vim-less#1]
Hi!
There seems to be a problem with the syntax color for some attributes (padding, border, font-size, left, right).
Keep up the good work!

**Comment by luishdez** [comment]
@klingberg Thanks for reporting! I'm checking that. Do you have the vim-css3-syntax plugin installed?

**Comment by klingberg** [comment]
Yes, I have the vim-css3-syntax plugin installed. I have tried a few different versions of the vim-less plugin and they have all had the same problem with the syntax color.

**Comment by luishdez** [comment]
Sorry for the delay…, I started to improve this plugin taking the base from other plugin, so it look that all of the LESS plugins available are buggy. I've started to write the syntax from scratch I hope it won't take to long

**Comment by klingberg** [comment]
No worries! Nice! Looking forward to it!
